.tool-versions

```diff
@@ -20,3 +20,4 @@ spotbugs 4.8.6
 # CouchDB tests
 elixir 1.17.3-otp-26
 python 3.12.7
+typst 0.14.0
```

Comment:
This is not a testing tool as the comment above would suggest.  Perhaps there could be another line of comment saying "documentation"?